microprot/scripts/split_search.py

```diff
@@ -453,6 +455,49 @@ def frag_size(hit):
     return len(subseq) - subseq.count('-')
 
 
+def correct_header_positions(header, new_start, new_end, out=sys.stderr):
+    """Prevents chaining of start-end positions.
+
+    Parameters
+    ----------
+    header : str
+        The existing fasta header string with start-end positions as a suffix.
```

Comment:
Optional: Looks like there is a very specific format required here.  It would be great if there was a `See Also` section to refer to the format being used here.  Or maybe even an example docstring will do.